lib/migration.js

```diff
@@ -80,39 +68,39 @@ module.exports = function(DB2iSeries) {
     });
   };
 
-  DB2iSeries.prototype.showFields = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{showFields()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.showFields = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{showFields()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.showIndexes = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{showIndexes()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.showIndexes = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{showIndexes()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.autoupdate = function(models, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{autoupdate()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.autoupdate = function(models, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{autoupdate()}} is not currently supported.')));
+//     });
+//   };
 
   DB2iSeries.prototype.isActual = function(models, cb) {
     process.nextTick(function() {
       return cb(Error(g.f('{{isActual()}} is not currently supported.')));
     });
   };
 
-  DB2iSeries.prototype.alterTable = function(model, fields, indexes, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{alterTable()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.alterTable = function(model, fields, indexes, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{alterTable()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.getTableStatus = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{getTableStatus()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.getTableStatus = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{getTableStatus()}} is not currently supported.')));
+//     });
+//   };
```

Comment:
Kind of hard to tell now that I look at it in more detail.  I was expecting autoupdate among other functions to be common and to be in loopback-ibmdb but they aren't.  I'm going to uncomment all these.  I don't think there are migration tests for this connector yet so it should be fine to ship with this but if I can get it working quickly I'll likely push changes to this as well as IBMDB and add the migration tests.